resources/qmk_layouts/sofle_rotated.json

```
@@ -0,0 +1,84 @@
+{
+    "url": "https://github.com/josefadamcik/SofleKeyboard",
+    "layouts": {
+        "LAYOUT": {
+            "layout": [
+                {"x": 0,  "y": 0.625},
+                {"x": 1,  "y": 0.625},
+                {"x": 2,  "y": 0.125},
+                {"x": 3,  "y": 0},
+                {"x": 4,  "y": 0.125},
+                {"x": 5,  "y": 0.25},
+                
+                {"x": 10, "y": 0.25},
+                {"x": 11, "y": 0.125},
+                {"x": 12, "y": 0},
+                {"x": 13, "y": 0.125},
+                {"x": 14, "y": 0.625},
+                {"x": 15, "y": 0.625},
+                
+                
+                {"x": 0,  "y": 1.625},
+                {"x": 1,  "y": 1.625},
+                {"x": 2,  "y": 1.125},
+                {"x": 3,  "y": 1},
+                {"x": 4,  "y": 1.125},
+                {"x": 5,  "y": 1.25},
+                
+                {"x": 10,  "y": 1.25},
+                {"x": 11, "y": 1.125},
+                {"x": 12, "y": 1},
+                {"x": 13, "y": 1.125},
+                {"x": 14, "y": 1.625},
+                {"x": 15, "y": 1.625},
+                
+                
+                {"x": 0,  "y": 2.625},
+                {"x": 1,  "y": 2.625},
+                {"x": 2,  "y": 2.125},
+                {"x": 3,  "y": 2},
+                {"x": 4,  "y": 2.125},
+                {"x": 5,  "y": 2.25},
+                
+                {"x": 10, "y": 2.25},
+                {"x": 11, "y": 2.125},
+                {"x": 12, "y": 2},
+                {"x": 13, "y": 2.125},
+                {"x": 14, "y": 2.625},
+                {"x": 15, "y": 2.625},
+                
+                
+                {"x": 0,  "y": 3.625},
+                {"x": 1,  "y": 3.625},
+                {"x": 2,  "y": 3.125},
+                {"x": 3,  "y": 3},
+                {"x": 4,  "y": 3.125},
+                {"x": 5,  "y": 3.25},
+                
+                {"x": 6.125,  "y": 3.25},
+                
+                {"x": 8.875,  "y": 3.25},
+                
+                {"x": 10, "y": 3.25},
+                {"x": 11, "y": 3.125},
+                {"x": 12, "y": 3},
+                {"x": 13, "y": 3.125},
+                {"x": 14, "y": 3.625},
+                {"x": 15, "y": 3.625},
+                
+                
+                {"x": 2,  "y": 4.125},
+                {"x": 3,  "y": 4},
+                {"x": 4,  "y": 4.125},
+                {"x": 5.15, "y": 4.4, "r": 22.5},
+                {"x": 6.25, "y": 4.45, "h": 1.5, "r": 30},
+                
+                {"x": 8.75, "y": 4.45, "h": 1.5, "r": -30},
+                {"x": 9.85, "y": 4.4, "r": -22.5},
```

```suggestion
                {"x": 5.15, "y": 4.4, "r": 22.5, "rx": 5.65, "ry": 4.9},
                {"x": 6.25, "y": 4.45, "h": 1.5, "r": 30, "rx": 6.75, "ry": 5.2},

                {"x": 8.75, "y": 4.45, "h": 1.5, "r": -30, "rx": 9.25, "ry": 5.2},
                {"x": 9.85, "y": 4.4, "r": -22.5, "rx": 10.35, "ry": 4.9},
```
I try to add `rx` `ry` so we preserve compatibility with other tools which typically don't assume rotation around key center.